baselines/NER/docile_inference_NER_multilabel.py

```diff
@@ -0,0 +1,968 @@
+import argparse
+import dataclasses
+import json
+import os
+from datetime import datetime
+from pathlib import Path
+
+import numpy as np
+import torch
+from helpers import FieldWithGroups, show_summary
+from my_roberta_multilabel import MyXLMRobertaMLForTokenClassification
+from tqdm import tqdm
+from transformers import AutoConfig, AutoTokenizer
+
+from docile.dataset import BBox, Dataset
+from docile.evaluation.evaluate import evaluate_dataset
+
+# from my_roberta import MyXLMRobertaForTokenClassification
+# from my_bert import MyBertForTokenClassification
+
+
+# KILE classes
+KILE_CLASSES = [
+    "account_num",
+    "amount_due",
+    "amount_paid",
+    "amount_total_gross",
+    "amount_total_net",
+    "amount_total_tax",
+    "bank_num",
+    "bic",
+    "currency_code_amount_due",
+    "customer_billing_address",
+    "customer_billing_name",
+    "customer_delivery_address",
+    "customer_delivery_name",
+    "customer_id",
+    "customer_order_id",
+    "customer_other_address",
+    "customer_other_name",
+    "customer_registration_id",
+    "customer_tax_id",
+    "date_due",
+    "date_issue",
+    "document_id",
+    "iban",
+    "order_id",
+    "payment_terms",
+    "tax_detail_gross",
+    "tax_detail_net",
+    "tax_detail_rate",
+    "tax_detail_tax",
+    # 'variable_symbol',
+    "payment_reference",
+    "vendor_address",
+    "vendor_email",
+    "vendor_name",
+    "vendor_order_id",
+    "vendor_registration_id",
+    "vendor_tax_id",
+]
+
+# LIR classes
+LIR_CLASSES = [
+    "line_item_amount_gross",
+    "line_item_amount_net",
+    "line_item_code",
+    "line_item_currency",
+    "line_item_date",
+    "line_item_description",
+    "line_item_discount_amount",
+    "line_item_discount_rate",
+    "line_item_hts_number",
+    "line_item_order_id",
+    "line_item_person_name",
+    "line_item_position",
+    "line_item_quantity",
+    "line_item_tax",
+    "line_item_tax_rate",
+    "line_item_unit_price_gross",
+    "line_item_unit_price_net",
+    "line_item_units_of_measure",
+    "line_item_weight",
+]
```

Comment:
You are right, I am happy to do that, there will be just one tiny difference, KILE_CLASSES will have a slightly different order (due to `payment_reference` not being in alphabetical order in your case as it replaced `variable_symbol`). Could that somehow affect results/trainings?

Reply:
It should not have any impact on the trained models, because the classes should be alphabetically ordered inside the model (see e.g. https://github.com/rossumai/docile/blob/fb91d582b9466f8fc5f3cedd922e160856390938/baselines/NER/docile_train_NER_multilabel.py#L857)

Reply:
I see, great! It also definitely should not impact inference (there the order is not used at all).